ee/vellum_ee/workflows/display/nodes/tests/test_base_node_display.py

```diff
@@ -46,3 +50,35 @@ def test_get_id(node_info):
 
     assert node_display().node_id == expected_id
     assert node_display.infer_node_class().__id__ == expected_id
+
+
+def test_serialize_condition__accessor_expression():
+    # GIVEN a node with an accessor expression in a Port
+    class MyNode(BaseNode):
+        class Ports(BaseNode.Ports):
+            foo = Port.on_if(ConstantValueReference({"hello": "world"})["hello"])
+
+    # WHEN we serialize the node
+    node_display_class = get_node_display_class(MyNode)
+    data = node_display_class().serialize(WorkflowDisplayContext())
```

Comment:
I went back and forth on supporting `node_display_class().serialize()`. I don't like how python's default initializers could be reusable